addons/website/views/snippets/snippets.xml

```diff
@@ -484,7 +484,7 @@
     <!-- Mobile display options -->
     <div data-option-name="showMobile" data-selector="section .row > div"
          data-exclude=".s_col_no_resize.row > div">
-        <we-checkbox string="Shown On Mobile" data-select-class="d-none d-md-block|" data-no-preview="true"/>
+        <we-checkbox string="Shown On Mobile" data-select-class="o_we_hidden_md_down|" data-no-preview="true"/>
```

Comment:
We could do that but it is not a stable change. If you do this, people which use the option and try to change the value later won't be able to do it. On top of that "d-md-block" meant "visible starting from md"... and you change that into hidden up to including md... so you also change the behavior for the md breakpoint :/

To fix this issue properly for stable and actually avoid creating an extra css rule even for master, use a custom JS option. Instead of data-select-class, create a dedicated method which will be in charge of checking and knowing if d-md-block or d-md-flex has to be used.